net-gateway/src/endpoints/pcap_files.rs

```diff
@@ -0,0 +1,53 @@
+use actix_web::post;
+use actix_web::web;
+use actix_web::HttpRequest;
+use actix_web::HttpResponse;
+use actix_web::Responder;
+use actix_multipart::Multipart;
+use futures::StreamExt;
+use futures::TryStreamExt;
+use net_agent_api::api::data_packet::DataPacketDTO;
+use net_core_api::envelope::envelope::Envelope;
+use net_core_api::typed_api::Typed;
+use net_core_api::encoder_api::Encoder;
+use net_token_verifier::fusion_auth::fusion_auth_verifier::FusionAuthVerifier;
+
+use crate::core::quinn_client_endpoint_manager::QuinnClientEndpointManager;
+use crate::{authorization, config::Config};
+
+#[post("/pcap_files")]
```

Comment:
It should probably be named like `insert_..`. I think it more appropriate.

Reply:
No, we shouldn't call it like that, it is a bad practice in REST

Reply:
Why is it?